scripts/wipe_sensitive_info_db.py

```diff
@@ -6,22 +6,32 @@
 from absl import app
 from absl import flags
 import click
-import sqlite3
+import os
+from sqlalchemy import create_engine
 
+from icubam.db.store import StoreFactory
 from icubam.db.wipe import wipe_db
 
 flags.DEFINE_boolean("force", False, "Do not prompt user confirmation.")
 flags.DEFINE_boolean(
   "keep_beds", False, "Whether to keep bed occupation data."
 )
+flags.DEFINE_boolean("reset_admin", False, "Reset admin information")
+flags.DEFINE_string("admin_email", None, "Admin email (if resetting admins)")
+flags.DEFINE_string(
```

Comment:
In the typical use case, what's this admin password? Is it a fake password for the wiped DB, or something more real/sensitive? If the former, leaking the env variable into it feel strange, if the latter, we should probably prompt for it rather than take it as a command line argument.

Reply:
I had two use cases in mind: quick checks on a dev machine (fake password) and staging server (stronger password even though there should be no sensitive info in the DB, it would probably be stored in the CI secrets hence the environment variable)

Reply:
Ok, fair enough. We can hope users know what they are doing :)